### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by xuallenx** [CSC207-2022F-UofT/course-project-macrosoft#61] [Announcements]
As we are nearing the final steps in our project, I would like to further bring everyone's attention to Yinuo's previous discussion post. Regarding our logo or application name, feel free to discuss below and give suggestions that you see fit.  We will then conduct a poll under discussions to finalize a name and logo!

**Comment by inorrr** [comment]
Up Up!

**Comment by KaiwenYangUT** [comment]
Here's my imagination:
App name: eTruck, eat@UT, TimeSaver, Ueats, UeaT...
:)
